### PR TITLE
fix: add disk limit to pb actors, fix dockerfiles

### PR DESCRIPTION
**Pull request by MasterPtato**
Fixes RVT-4193
Fixes RVT-4202

### CODE REVIEW

**Comment by MasterPtato** [comment]
* **#1469** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/1469?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/1469?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: packages/infra/client/container-runner/src/container.rs [hunk 1]
```diff
@@ -32,8 +32,8 @@ pub fn run(
 		.context("empty `actor_path`")?
 		.to_string_lossy()
 		.to_string();
-	let oci_bundle_path = actor_path.join("oci-bundle");
-	let oci_bundle_config_json = oci_bundle_path.join("config.json");
+	let fs_path = actor_path.join("fs");
```
Comment (NathanFlurry) [inline packages/infra/client/container-runner/src/container.rs:35]: nit: technically the fs root is in `oci-bundle/root`, so naming this "fs" is a bit confusing for me.

thoughts on moving it to `fs/oci-bundle`? also gives us flexibility to add more files in the future.

File: packages/infra/client/isolate-v8-runner/src/isolate.rs [hunk 2]
```diff
@@ -146,8 +146,8 @@ pub async fn run_inner(
 
 	tracing::info!(?actor_id, "isolate kv initialized");
 
-	// Should match the path from `Actor::download_image` in manager/src/actor/setup.rs
-	let entrypoint = actor_path.join("js-bundle").join("index.js");
+	// Should match the path from `Actor::make_fs` in manager/src/actor/setup.rs
+	let entrypoint = actor_path.join("fs").join("index.js");
```
Comment (NathanFlurry) [inline packages/infra/client/isolate-v8-runner/src/isolate.rs:150]: similar to above, imo we should have a "source dir" or something for the actor. thoughts?

File: packages/infra/client/manager/src/actor/partial_oci_config.rs [hunk 3]
```diff
@@ -10,6 +10,15 @@ pub struct PartialOciConfig {
 pub struct PartialOciConfigProcess {
 	pub args: Vec<String>,
 	pub env: Vec<String>,
-	pub user: String,
+	pub user: PartialOciConfigUser,
```
Comment (NathanFlurry) [inline packages/infra/client/manager/src/actor/partial_oci_config.rs:13]: oops mb thx

**Review by NathanFlurry — CHANGES_REQUESTED** [review]

File: packages/infra/client/isolate-v8-runner/src/isolate.rs (hunk 2 above)
Comment (MasterPtato) [reply packages/infra/client/isolate-v8-runner/src/isolate.rs:150]: This is the entire file system for the isolate though, do you want it to be fs/src?

File: packages/infra/client/container-runner/src/container.rs (hunk 1 above)
Comment (MasterPtato) [reply packages/infra/client/container-runner/src/container.rs:35]: Its sort of a double embedding, i didnt bother trying to make a custom fs for the bundle's root folder and instead put it inside the `fs` folder. so the `fs` folder is in fact a file system, there just happens to be an oci fs inside

Comment (NathanFlurry) [reply packages/infra/client/container-runner/src/container.rs:35]: roger, i misunderstood this. i thought that we were using this as a "staging folder" for all the untar/umoci/skopeo operations. looks like it's just for running. disregard.

File: packages/infra/client/isolate-v8-runner/src/isolate.rs (hunk 2 above)
Comment (NathanFlurry) [reply packages/infra/client/isolate-v8-runner/src/isolate.rs:150]: see above

**Review by NathanFlurry — APPROVED** [review]

**Pull request closed without merging**